src/test/java/org/minions/devfund/noemiguzman/moviedatabase/MovieTest.java

```diff
@@ -0,0 +1,74 @@
+package org.minions.devfund.noemiguzman.moviedatabase;
+
+import java.util.ArrayList;
+
+/**
+ * class to test movie.
+ */
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *  Class to test Movie.
+ */
+
+public class MovieTest {
+    private Actor actor1;
+    private Actor actor2;
+    private Movie movie1;
+    private Movie movie2;
+    private ArrayList<Movie> movieList1;
+    private ArrayList<Movie> movieList2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "movieList2" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)